sdk/python/feast/repo_operations.py

```diff
@@ -132,3 +137,82 @@ def registry_dump(repo_config: RepoConfig):
         print(entity)
     for table in registry.list_feature_tables(project=project):
         print(table)
+
+
+def cli_check_repo(repo_path: Path):
+    config_path = repo_path / "feature_store.yaml"
+    if not config_path.exists():
+        print(
+            f"Can't find feature_store.yaml at {repo_path}. Make sure you're running this command in an initialized feast repository."
+        )
+        sys.exit(1)
+
+
+def init_repo(repo_path: Path, minimal: bool):
+
+    repo_config = repo_path / "feature_store.yaml"
+
+    if repo_config.exists():
+        print("Feature repository is already initalized, nothing to do.")
+        sys.exit(1)
+
+    project_id = "proj" + "".join(
+        random.choice(string.ascii_lowercase + string.digits) for _ in range(5)
+    )
+
+    if minimal:
+        repo_config.write_text(
+            dedent(
+                f"""
+        project: {project_id}
+        metadata_store: /path/to/metadata.db
+        provider: local
+        online_store:
+            local:
+                path: /path/to/online_store.db
+        """
+            )
+        )
+        print(
+            "Generated example feature_store.yaml. Please edit metadata_store"
```

Comment:
Why do we need to edit the metadata store location before running apply?

Reply:
Is there any reason we cant use a local path? `metadata.db`?

Reply:
We could but you'll have to edit `online_store` there anyway. The idea here, unlike non-minimal init, you know what you're doing, you'll want to edit the config file anyway. Maybe change provider too. So I don't want to provide "example" values that may not be what you want but are valid.

Reply:
Ok, I see what you're trying to do. I just don't know how I feel about that. If the user knows what they are doing then presumably they can create a new `.yaml` by hand. I guess its not a big deal right now.